src/timer/Timer.js

```diff
@@ -1,17 +1,21 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+// Refactor to use ES6 classes
+module.exports = class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function() {
-  var instance = this;
-  var timerInterval = setInterval(function() {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    this.timer = setInterval(() => {
+      if (this.seconds === 0) {
+      // eslint-disable-next-line no-console
+        console.log('Done!');
+        clearInterval(this.timer);
+        return;
```

Comment:
Note that returning on this line slightly changed the behavior of the timer. The 0 will never print. Not a huge deal, but make sure to keep edge cases in mind when refactoring code.